boreas/boreas_io.h

```diff
@@ -34,11 +34,9 @@ put_host_on_queue (kb_t, char *);
 void
 put_finish_signal_on_queue (void *);
 
-void
-realloc_finish_signal_on_queue(kb_t);
+void realloc_finish_signal_on_queue (kb_t);
```

Comment:
Not sure why you're moving these down when the others are up.  Is clang-format asking for it?

Reply:
I have just run clang-format and change it.